pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -506,7 +506,7 @@ type S3StorageProvider struct {
 	Provider S3StorageProviderType `json:"provider"`
 	// Region in which the S3 compatible bucket is located.
 	Region string `json:"region,omitempty"`
-	// Bucket in which to store the Backup.
+	// Bucket in which to store the backup data.
```

Comment:
We do not support specifying a prefix?

Reply:
What do you mean by specifying a bucket prefix and adding a random string to construct a complete bucket name?

Reply:
Users may want to store the backup file under a specific directory (prefix), though we have generated a prefix composed by cluster name and namespace.

Reply:
In this pr,  users must specify the bucket name, since backup is not the same as before. There is no cluster and namespace, so the bucket name will not be generated automatically.

Reply:
The bucket name is different from prefix. I'm wondering if we should provide another parameter `prefix`, so users can store backup file in a bucket under a specific directory.

Reply:
I have added one on https://github.com/pingcap/tidb-operator/pull/1280